error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/BugCheckerRules.java

```diff
@@ -0,0 +1,30 @@
+package tech.picnic.errorprone.refasterrules;
+
+import com.google.errorprone.BugCheckerRefactoringTestHelper;
+import com.google.errorprone.BugCheckerRefactoringTestHelper.FixChooser;
+import com.google.errorprone.BugCheckerRefactoringTestHelper.FixChoosers;
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import tech.picnic.errorprone.refaster.annotation.OnlineDocumentation;
+
+/** Refaster rules related to BugChecker classes. */
+@OnlineDocumentation
+final class BugCheckerRules {
```

Comment:
Honestly not sure about the reach of this class 🤔 
I could also limit it to `BugCheckerRefactoringTestHelper`, but then it's also quite closed.
That would also impact docs and all.

Reply:
Hahaha, yeah I saw this class and hesitated, but felt like doing this was cleaner ;) 
Thanks for the added rule 💪